quantum/keymap_extras/keymap_spanish.h

```diff
@@ -117,7 +117,7 @@
 #define ES_CIRC S(ES_GRV)  // ^ (dead)
 #define ES_ASTR S(ES_PLUS) // *
 // Row 3
-#define ES_DIAE S(ES_GRV)  // ¨ (dead)
+#define ES_DIAE S(ES_ACUT)  // ¨ (dead)
```

Comment:
```suggestion
#define ES_DIAE S(ES_ACUT) // ¨ (dead)
```

Reply:
Also, looks like this bug is present in keymap_spanish_dvorak.h too.

Reply:
dvorak variant doesn't have `ES_DIAE`